### DISCUSSION TITLE
How to collapse all itens in dashboard (and maintain only that you clicked)

### DISCUSSION THREAD

**Discussion by lunegreiros** [ColorlibHQ/AdminLTE#3313] [Q&A] [answered]
Anyone have a ideia how to collapse (onclick event) all the itens from dashboard, and then open only that you are clicked? you have a class injection '.menu-open' on a `<li>` when clicked, but is not removeClass, when you clicked in another item. boths mantain the '.menu-open' class. I like to remove that class from all others itens in dashboard. regards.

**Comment by REJack** [comment, marked as answer]
Do you mean collapsing the sidebar menu items, like a accordion?

**Reply by lunegreiros** [reply]
exactly. like the demo (https://adminlte.io/themes/AdminLTE/index2.html) but my implementation doesnt close. see on: http://143.107.213.227/layout/mapa_tematico/index.html

**Reply by REJack** [reply]
You need to enable `accordion`, like this example:
```html
<ul data-widget="treeview" data-accordion="true">...</ul>
```

Like we use it on the Docs, https://adminlte.io/docs/3.0/javascript/treeview.html.

**Reply by lunegreiros** [reply]
Perfect! My regards.
